scripts/parseDownloadRequests.py

```diff
@@ -42,10 +41,10 @@ def main():
 
 def load_batch():
     paginator = s3_client.get_paginator('list_objects_v2')
-    page_iterator = paginator.paginate(Bucket= bucketName, Prefix= 'logs/946183545209/us-east-1/ump-pdf-repository/2024/03/13/')
+    page_iterator = paginator.paginate(Bucket= bucketName, Prefix=logPrefix)
     return page_iterator
 
-def parseInfo(logObject, requestRegex, referrerRegex, timeStampRegex, fileIDRegex):
+def parseInfo(logObject):
```

Comment:
I think for the purposes of the script I don't mind whether these values are fed in as parameters or not; the only thing I felt strongly about was not hard-coding e.g. the prefix when the function was called & that has been changed. 

I think this looks OK too, but if we do abstract this code in the future (for example, to ingest books from other presses) we might want to think about how this function signature should be shaped.